Source/Csla.Blazor/PropertyInfo.cs

```diff
@@ -16,8 +16,10 @@ namespace Csla.Blazor
   /// </summary>
   public class PropertyInfo : IPropertyInfo, INotifyPropertyChanged
   {
-    private const string TextSeparator = ", ";
-    
+    /// <summary>
+    /// Gets the value of the TextSeparator
+    /// </summary>
+    public string TextSeparator { get; }
```

Comment:
Why are we changing the default from ", " to " "? I would think we should keep the default the same?

Reply:
it was there in description that Default will be " ".

![image](https://github.com/MarimerLLC/csla/assets/56026691/fab37e14-ce17-49c2-9752-61327f82e3f8)

Reply:
My bad 🫢